nbconvert/exporters/latex.py

```diff
@@ -43,6 +43,10 @@ def _template_skeleton_path_default(self):
     template_extension = Unicode(".tplx").tag(config=True)
 
     output_mimetype = 'text/latex'
+    
+    def default_filters(self):
+        yield from super(TemplateExporter,self).default_filters()
```

Comment:
We should be passing in the *current* class here, i.e. `LatexExporter`, not `TemplateExporter`.

Python 3 improved on this, so you can just call `super().default_filters()` as we do in bookbook. But nbconvert still has to support Python 2 for now.